engine/apps/alerts/tasks/notify_user.py

```diff
@@ -42,7 +43,6 @@ def notify_user_task(
 
     countdown = 0
     stop_escalation = False
-    log_message = ""
```

Comment:
`log_message` is _only_ ever updated [here](https://github.com/grafana/oncall/pull/4635/files#diff-fa8513da06f09885842f55c35f819b5a14dd5e3b82ed8a5e1c4719bb64f1c447L137), but then it is never referenced/used anywhere